pkg/operator/sync.go

```diff
@@ -230,10 +265,13 @@ func (optr *Operator) waitForCustomResourceDefinition(resource *apiextv1beta1.Cu
 
 func (optr *Operator) waitForDeploymentRollout(resource *appsv1.Deployment) error {
 	return wait.Poll(deploymentRolloutPollInterval, deploymentRolloutTimeout, func() (bool, error) {
-		d, err := optr.deployLister.Deployments(resource.Namespace).Get(resource.Name)
+		//d, err := optr.deployLister.Deployments(resource.Namespace).Get(resource.Name)
```

Comment:
can we removed commented code. Why not deployLister

Reply:
that version issue happens with deployList. We can debug later. For now got this working by using kubeClient.AppsV1()